app/views/passenger_mailer/notify_active.haml

```diff
@@ -0,0 +1,9 @@
+Dear #{@passenger.name},
+
+Your account with the Special Transportation Service has been activated. You can receive rides #{('until ' + @passenger.rides_expire.strftime("%m/%d/%Y")) || 'permanently'}.
```

Comment:
This line will never say "You can receive rides permanently". The `||` operator will only evaluate the second clause if the first one evaluates to either `nil` or `false`.